src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs

```diff
@@ -110,11 +110,13 @@ public static class LogEventExtensions
         {
             if (logEvent == null) throw new ArgumentNullException("logEvent");
             if (logEvent.Exception == null) throw new ArgumentException("Must have an Exception", "logEvent");
-            
+
             var exceptionTelemetry = new ExceptionTelemetry(logEvent.Exception)
             {
                 SeverityLevel = logEvent.Level.ToSeverityLevel(),
+#if !NETSTANDARD1_6
```

Comment:
We've found the target framework monikers are a bit fragile to use in `#if` blocks; would it be possible to use:

```csharp
#if EXCEPTION_TELEMETRY_HANDLED_AT
...
```

and then `"buildOptions": { "define": ["EXCEPTION_TELEMETRY_HANDLED_AT"] }` in the two targets that support it?